google-cloud-pubsub/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Release History
 
+### 3.0.0 (2025-07-29)
+
+### ⚠ BREAKING CHANGES
+
+* The existing resource management API will be deprecated, and replaced with the gRPC autogenerated surface. It includes CRUDL operations for topics, subscriptions, snapshots, and schemas as well as IAM. All legacy handwritten resource management classes are now deleted and operations will be performed via the generated clients.
+* Rename Topic and Subscription to Publisher and Subscriber, which now serve as dedicated objects for sending and receiving messages.
+* Introduce subclasses for TopicAdmin & SubscriptionAdmin auto-generated clients ([#30744](https://github.com/googleapis/google-cloud-ruby/issues/30744))
```

Comment:
I don't think this is technically a breaking change either. And in fact we probably don't need to list it at all since these are covered by the top bullet point.